paris-processor/src/main/java/com/airbnb/paris/processor/framework/SkyKotlinFile.kt

```diff
@@ -0,0 +1,45 @@
+package com.airbnb.paris.processor.framework
+
+import com.squareup.kotlinpoet.FileSpec
+import java.io.File
+import javax.annotation.processing.ProcessingEnvironment
+
+
+internal abstract class SkyKotlinFile(
+    private val packageName: String,
+    private val name: String,
+    private val block: FileSpec.Builder.() -> Unit
+) {
+
+    fun build(): FileSpec {
+        return FileSpec.builder(packageName, name).run {
+            block()
+            build()
+        }
+    }
+
+    /**
+     * See [kaptOutputPath]
+     */
+    fun write(outputPath: String) {
+        build().writeTo(File(outputPath))
+    }
+}
+
+// This option will be present when processed by kapt, and it tells us where to put our
+// generated kotlin files
+// https://github.com/JetBrains/kotlin-examples/blob/master/gradle/kotlin-code-generation
```

Comment:
nit: the URL should be on a single line. Also why not use `/** ... */`?

Reply:
sure. legacy from an inline comment I copied from epoxy